### DISCUSSION TITLE
Image with notification

### DISCUSSION THREAD

**Discussion by Petran54** [djdd87/SynoAI#217] [Q&A]
Hi,
Thank you very much for a superb solution with SynoAI!

When recording is activated, I send a webhook to my Homey Pro 2023. I've come so far that I receive a notification about which camera has made the detection. But how do I include a snapshot that it has triggered on? Thus, I send a second webhook under action rules, after the start of recording. I'm not sufficiently knowledgeable about webhooks. I also don't know where to find the current image. Grateful for some help. The notification comes quickly that way, so I would like to keep that solution.

Webhook
GET
http://192.168.68.119/webhook?event=SS&tag=Garage

**Comment by djdd87** [comment]
Images are included by default when using a Webhook unless you tell it to not to. The field the image is sent in is called `image`. I have no experience of Homey Pro, so I don't know what it requires to pick up the image. If it needs a different field name, then you can change it in your config by setting the `ImageField` name.

**Reply by Petran54** [reply]
"Thank you for your response! When you say that an image is sent by default with the webhook, do you perhaps mean in your program? Considering that one could change the name in the config. Here, I am sending the webhook, as above, from Synology Surveillance Station which might be different.
There is a lot to study with webhooks, even then in Homey. I think I managed to find another solution where the Surveillance Station and its cameras are now set up as devices in Homey, and I can take a snapshot in the flow for each one. It won't be exactly the image it was triggered by, but it will have to suffice.
I'll have to read up a bit more as time goes on because it is useful. Thank you // Peter

**Reply by djdd87** [reply]
Yes, I'm referring the Webhook notification type in SynoAI. I'm not sure what you're asking? If you're sending a webhook request from SSS then that won't include a snapshot and that's nothing to do with SynoAI.

Is there any question here at all about SynoAI or how to do something with SynoAI?

**Reply by Petran54** [reply]
"I apologize for going a bit off-topic. But the initial idea was to receive a notification with the image that SynoAI analyzed through Deepstack, which apparently is included in the webhook that starts the recording. That I, in the same rule, then made a webhook to my Homey and was after the URL to this image or in another way. It would have been desirable to have a colored frame around the object, etc."

**Reply by djdd87** [reply]
That should be the image that's sent with the SynoAI webhook notification.

**Reply by Petran54** [reply]
Thank you for your responses. I might look into a solution where I send a webhook from SynoAI directly to Homey. You can close this, and I will think it over...
